scRNA-seq-advanced/setup/intro-diff-expression-exploration.Rmd

```diff
@@ -0,0 +1,800 @@
+# here we will actually put all of the significant genes so then we can plot any of them 
+sig_expressed_genes <- deseq_df %>%
+  dplyr::filter(pvalue < 1e-5,
+                abs(log2FoldChange) > 2) %>%
+  dplyr::pull(ensembl_id)
+
+sig_genes_counts <- logcounts(combined_sce[sig_expressed_genes,]) %>%
+  as.matrix() %>%
+  t() %>%
+  as.data.frame() %>%
+  dplyr::mutate(cell_id = colnames(combined_sce))
+  
+combined_coldata_df <- as.data.frame(colData(combined_sce)) %>%
+  dplyr::left_join(sig_genes_counts, by = "cell_id")
+
+# add back modified coldata containing gene expression for sig genes 
+colData(combined_sce) <- DataFrame(combined_coldata_df, row.names = combined_coldata_df$cell_id)
+```
+
+The first plot we will make is with SOX5 which is found in both `edgeR` and `DESeq` 
+
+```{r}
+# filter to just myoblast cells and remove any NA's before plotting
+myoblast_combined_sce <- combined_sce[, which(combined_sce$celltype_broad == "Tumor_Myoblast")]
+
+# first look at just ARMS vs. ERMS 
+scater::plotReducedDim(myoblast_combined_sce,
+                       dimred = "fastMNN_UMAP",
+                       colour_by = "ENSG00000134532", #SOX5
+                       point_size= 0.5,
+                       point_alpha = 0.4,
+                       other_fields = "diagnosis_groups") +
+  facet_wrap(~ diagnosis_groups, nrow = 3)
+```
+
+```{r}
+# now let's look at the same gene across each sample 
+scater::plotReducedDim(myoblast_combined_sce,
+                       dimred = "fastMNN_UMAP",
+                       colour_by = "ENSG00000134532", #SOX5
+                       point_size= 0.5,
+                       point_alpha = 0.4,
+                       other_fields = c("diagnosis_groups", "batch")) +
+  facet_wrap(~ diagnosis_groups + batch, nrow = 3, dir = 'v')
+```
+
+What about some genes that are differentially expressed in only `DESeq` but not `edgeR`? 
+
+```{r}
+# pick out the top genes that are identified as DE in only DESeq and plot those 
+combined_results %>%
+  dplyr::filter(significance == "DESeq") %>%
+  dplyr::arrange(DESeq_LogFC) 
+```
+
+```{r}
+# top downregulated gene 
+scater::plotReducedDim(myoblast_combined_sce,
+                       dimred = "fastMNN_UMAP",
+                       colour_by = "ENSG00000182197", #EXT1
+                       point_size= 0.5,
+                       point_alpha = 0.4,
+                       other_fields = c("diagnosis_groups", "batch")) +
+  facet_wrap(~ batch + diagnosis_groups, nrow = 3)
+```
+
+```{r}
+scater::plotReducedDim(myoblast_combined_sce,
+                       dimred = "fastMNN_UMAP",
+                       colour_by = "ENSG00000161671", #EXT1
+                       point_size= 0.5,
+                       point_alpha = 0.4,
+                       other_fields = c("diagnosis_groups", "batch")) +
+  facet_wrap(~ batch + diagnosis_groups, nrow = 3)
+```
+
+
+## Differential expression of a secondary cell type 
```

Comment:
```suggestion
## Differential expression of a second cell type 
```

"secondary" sounds to me like we're looking at nested/hierarchical cell types instead of just "a different cell type."